src/EditorFeatures/Test2/IntelliSense/SignatureHelpControllerTests.vb

```diff
@@ -94,6 +94,40 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense
         End Sub
 
         <WpfFact>
+        Public Sub DownKeyShouldNotBlockOnModelComputation()
```

Comment:
Do we need equivalent tests that verify that we *do* block once it has been presented?